cmd/up.go

```diff
@@ -70,7 +143,21 @@ func ClusterUp(
 
 	log.Infof(ctx, "Building Kubernetes cluster")
 	var APIURL, caCrt, clientCert, clientKey string
-	kubeCluster, err := cluster.ParseCluster(ctx, rkeConfig, clusterFilePath, configDir, dockerDialerFactory, localConnDialerFactory, k8sWrapTransport)
+
+	// is tehre any chance we can store the cluster object here instead of the rke config ?
```

Comment:
Do we intend to leave these comments around?

Reply:
removing this comment